frontend/packages/ceph-storage-plugin/integration-tests/views/pvc.view.ts

```diff
@@ -0,0 +1,95 @@
+import { browser, by, element, ExpectedConditions as until, $ } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { PVC_STATUS } from '../utils/consts';
+
+export const selectItemFromDropdown = async (item, dropdownElement) => {
+  await click(dropdownElement);
+  await click($(`#${item}-link`));
+};
+
+// create pvc
+export const namespaceDropdown = $$('[class="pf-c-dropdown__toggle pf-m-plain"]').get(0);
+export const storageclassDropdown = $('#storageclass-dropdown');
+export const inputPVCName = $('#pvc-name');
+export const selectAccessMode = (accessMode) => $(`input[value=${accessMode}]`);
+export const inputPVCSize = $('[name=requestSizeValue]');
+export const sizeUnitsDropdown = $('[data-test-id=dropdown-button]');
+
+// pvc details
+export const pvcName = element(by.xpath(".//dt[text()='Name']/following-sibling::dd[1]"));
```

Comment:
Well... not sure what to do here, xpath should be avoided, but I don't see other way to select the elements on PVC page. Do you think you could work with developers to annotate UI elements you're selecting with `data-test-id` attributes so that more simple and robust selectors could be used?

Reply:
Done